script.js

```diff
@@ -25,3 +26,29 @@ con.addEventListener('click', () => {
 por.addEventListener('click', () => {
   navexpand.classList.toggle('active');
 });
+
+const button0 = document.querySelector('.button0');
+const jsdetails = document.querySelector('.jsdetails');
+const xman = document.querySelector('.xman');
+
+button0.addEventListener('click', () => {
+  button0.classList.toggle('active');
+  jsdetails.classList.toggle('active');
+});
+
+xman.addEventListener('click', () => {
+  jsdetails.classList.toggle('active');
+});
+
+const deskbtn = document.querySelector('#deskbtn');
+const detaildsk = document.querySelector('.detaildsk');
+const desk = document.querySelector('.desk');
+
+deskbtn.addEventListener('click', () => {
+  deskbtn.classList.toggle('active');
+  detaildsk.classList.toggle('active');
+});
+
+desk.addEventListener('click', () => {
+  detaildsk.classList.toggle('active');
+});
```

Comment:
**[REQUIRED CHANGES]**

- [ ]  Hey, According to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), the [**works section**](https://github.com/0zedd/Portfolio/blob/286e01d3ea6718ca9ea6004db033c1ad6707a5a0/index.html#L60-L268) should be added dynamically using javascript. please kindly fix this issue.0
- [ ] Also, kindly ensure the design matches the Figma design. As of now, it's having some unexpected behavior that does not improve the user experience. Please, kindly fix this issue.
- [ ] The modal only shows when the first project card button is clicked. Please, ensure all the buttons show up in the pop window when clicked.

**[OPTIONAL]**  Your portfolio is having a horizontal scrolling effect which is not a great experience for the users. Please, kindly fix this issue to improve the user experience.

**Please, check the video below to see you pop up in action**



https://user-images.githubusercontent.com/59397159/175346178-06ff6005-1b8b-4b48-9416-46627c58b18e.mp4